### DISCUSSION TITLE
bootstrap script for RHEL platform

### DISCUSSION THREAD

**Discussion by tjyang** [tursodatabase/libsql#84] [Ideas]
Hi All
I have the first draft of bootstrap script for RHEL OS.
It is in less intrusive contrib sub directory, 
https://github.com/libsql/libsql/compare/main...learnrust:libsql:main

- After finish the precheck of needed  rpms in **contrib/bootstrap.RHEL.sh**, sqlite3 and libsql are generated. 

```
[me@fedora01d libsql]$ contrib/bootstrap.RHEL.sh;ls -lrt
<snipped>
-rw-r--r--. 1 me me     271 Dec 10 12:56 sqlite3.pc
-rwxr-xr-x. 1 me me  259675 Dec 10 12:56 libtool
-rw-r--r--. 1 me me   66539 Dec 10 12:56 config.log
-rwxr-xr-x. 1 me me 6578144 Dec 10 12:56 sqlite3
-rwxr-xr-x. 1 me me 6578144 Dec 10 12:56 libsql
[me@fedora01d libsql]$
```

- Let me know if this useful enough to submit a PR to this project.

**Comment by psarna** [comment]
Yes it is! Thanks for willing to contribute, please submit your change as a PR and we'll get it through review and CI.

**Reply by tjyang** [reply]
https://github.com/libsql/libsql/pull/85 submitted.
